v1/api/swagger/paths/hosts@{hostId}.yaml

```diff
@@ -34,7 +34,7 @@ put:
   description: >-
     Updates the Host with the matching ID.  Update is permitted only if the host
     is in the 'Ready' or 'Connection Updating Failed' state.  Only the Host
-    'Description', 'Networks', 'NetworkForDefaultRoute', and 'ISCSIConfig:InitiatorName'
+    'Description', 'Networks', 'NetworkForDefaultRoute', 'NetworkUntagged' and 'ISCSIConfig:InitiatorName'
```

Comment:
Updatable fields needs to be marked as required - https://github.com/HewlettPackard/hpegl-metal-client/blob/057956e724ccec3c9b2d906d8757c50ec5b90642/v1/api/swagger/components/schemas/Host.yaml#L4

Reply:
https://github.com/hpe-hcss/quake/blob/1e4d12bd88021a8e3a024572d618600263107e45/portal/pkg/host/extra.go#L224 If it is not sent with the update request, then it will get reset. So, we have been following patterns where updatable fields are marked as required.